numpy/f2py/__init__.pyi

```diff
@@ -1,18 +1,40 @@
-from typing import Any, List
+import os
+import subprocess
+from typing import Any, List, Iterable, Dict, overload
+from typing_extensions import TypedDict, Literal as L
 
-from numpy.f2py import (
-    f2py_testing as f2py_testing,
-)
+class _F2PyDictBase(TypedDict):
+    csrc: List[str]
+    h: List[str]
+
+class _F2PyDict(_F2PyDictBase, total=False):
+    fsrc: List[str]
+    ltx: List[str]
 
 __all__: List[str]
+__path__: List[str]
+
+def run_main(comline_list: Iterable[str]) -> Dict[str, _F2PyDict]: ...
```

Comment:
For reference: `_F2PyDict` is a normal dictionary that _always_ contains the `csrc` and `h` keys, 
and may or may not contain `fsrc` and/or `ltx`.